api/spot.yaml

```diff
@@ -617,4 +629,29 @@ paths:
     $ref: services/ocean/cd/paths/rollout.yaml
   /ocean/cd/rollout/{rolloutId}/externalVerification:
     $ref: services/ocean/cd/paths/rolloutExternalVerification.yaml
-
+  /ocean/spark/cluster:
+    $ref: services/ocean/spark/paths/clusters.yaml
+  /ocean/spark/cluster/{clusterId}:
+    $ref: services/ocean/spark/paths/cluster.yaml
+  /ocean/spark/app:
+    $ref: services/ocean/spark/paths/applications.yaml
+  /ocean/spark/cluster/{clusterId}/app:
+    $ref: services/ocean/spark/paths/clusterApplications.yaml
+  /ocean/spark/cluster/{clusterId}/app/{applicationId}:
+    $ref: services/ocean/spark/paths/clusterApplication.yaml
+  /ocean/spark/cluster/{clusterId}/app/{applicationId}/logs/live:
+    $ref: services/ocean/spark/paths/clusterApplicationLogsLive.yaml
+  /ocean/spark/cluster/{clusterId}/app/{applicationId}/kubeEvents/live:
+    $ref: services/ocean/spark/paths/clusterApplicationKubeEventsLive.yaml
+  /ocean/spark/configTemplate:
```

Comment:
@HHK1 this endpoint is not exposed in the optimizer. If you agree, can your team add it to the optimizer?